testplan/web_ui/testing/public/index.html

```diff
@@ -4,9 +4,16 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/testplan-logo.png">
-    <link href="//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css"
```

Comment:
Well the reason I edited this at all is because there were warnings in the dev console about CORS problems with the original tag, and the bootstrap site recommended this CDN.

Maybe I could upgrade it to 4.7 in another PR?